WordPress/Classes/ViewRelated/Blog/SiteManagement/SiteTagsViewController.swift

```diff
@@ -0,0 +1,266 @@
+import UIKit
+
+final class SiteTagsViewController: UITableViewController, NSFetchedResultsControllerDelegate {
+    private struct TableConstants {
+        static let cellIdentifier = "TitleBadgeDisclosureCell"
+        static let accesibilityIdentifier = "SiteTagsList"
+        static let numberOfSections = 1
+    }
+    private let blog: Blog
+
+    fileprivate let noResultsView = WPNoResultsView()
+
+    fileprivate lazy var context: NSManagedObjectContext = {
+        return ContextManager.sharedInstance().newMainContextChildContext()
```

Comment:
I'm curious about using a child context vs the main context.

Reply:
Oops, so am I! 😊 Sorry!